src/interfaces/cmodels.py

```diff
@@ -0,0 +1,65 @@
+from datetime import datetime
+
+from pymongo.collection import Collection
+from pymongo.mongo_client import MongoClient
+
+from src.models.db.cmodels import (
+    CompartmentalModelBase,
+    CompartmentalModelEnum
+)
+from src.models.db.cmodels import CompartmentalModel
+from src.interfaces.crud import MongoCRUD
+
+
+class CModelsInterface:
+    def __init__(
+        self,
+        db_connection: MongoClient,
+        collection: Collection
+    ) -> None:
+        self.crud = MongoCRUD(db_connection, collection)
+
+    def insert_one_cmodel_document(self, model: CompartmentalModelBase):
+
+        id_dict = {'_id': model.name.encode('utf-8').hex()}
+
+        cmodel_document = CompartmentalModel(
+            id=id_dict['_id'],
+            inserted_at=datetime.now(),
+            updated_at=datetime.now(),
+            **model.dict()
+        ).dict(by_alias=True)
+
+        existent_model = self.crud.read(id_dict)
+
+        if existent_model:
+            pruned_existent_model = \
+                CModelsInterface._prune_db_document(
+                    existent_model
+                )
+            if pruned_existent_model == model.dict(by_alias=True):
+                # TODO: log cmodel exists
+                print(f'Cmodel exists: {model.name}')
```

Comment:
Never write `print` function in production code, this is a bad practice in APIs, just use in your local development.

Reply:
True, David. That's why we have the TODO label. The idea is to change print -> log